### ISSUE TITLE
MSW dark mode status bar thicker borders

### ISSUE THREAD

**Issue by marekr** [wxWidgets/wxWidgets#23621]
This is using master from d80887f3

You can see that the border appears twice as thick as it should be
![image](https://github.com/wxWidgets/wxWidgets/assets/1838013/a77ad496-5d5c-4821-92f2-bbf90188ed9d)

![image](https://github.com/wxWidgets/wxWidgets/assets/1838013/e3d8ef09-0a20-456a-b6ad-9838f9235397)

vs non-dark mode
![image](https://github.com/wxWidgets/wxWidgets/assets/1838013/7e653b92-c4d8-4c7e-b5be-f4b52a52416d)

**Comment by vadz** [comment]
Yes, I've noticed this too but hoped it wouldn't result in too many complaints... The problem is that this is how `ExplorerStatusBar` theme draws the status bar and this was the only theme I could find which uses dark colours for it. So if we can't live with this double border we need to switch to drawing the status bar entirely ourselves (at least in the dark mode). Considering that we already draw the gripper ourselves, it's probably not going to be such a huge deal but I still prefer to let the system draw as much as possible as this isolates us better from any future changes, e.g. if they start using skeuomorphic status bars in Windows 12 the current version would use the new look automatically while custom drawn one wouldn't.
